tests/unit/literaturesuggest/test_literaturesuggest_tasks.py

```diff
@@ -65,15 +98,16 @@ def test_formdata_to_model_populates_schema(u, ui):
     obj = StubObj(data, extra_data)
 
     expected = 'https://localhost:5000/schemas/records/hep.json'
-    result = formdata_to_model(obj, formdata)
+    with app.app_context():
+        result = formdata_to_model(obj, formdata)
```

Comment:
That's the thing: if you required the `app` fixture you already have an application context, so you can just
```python
result = formdata_to_model(obj, formdata)
```